spec/rails_helper.rb

```diff
@@ -75,4 +75,16 @@
     with.test_framework :rspec
     with.library :rails
   end
+
+  class RequestObj
```

Comment:
Cool you draw @andela-oakinniranye or @andela-oojewale attention to this. Or maybe I will try it out myself.
But in the spirit of keeping things `DRY` we really need to abstract this out given that token generation will be vital to feature testing.